### DISCUSSION TITLE
Ua2baro/Adv_flux_2 term sign

### DISCUSSION THREAD

**Discussion by silver-fox-9244** [csyhuang/hn2016_falwa#125] [Q&A] [answered]
Hello

1. I was wondering whether the term ua2baro as output from _compute_lwa_and_barotropic_fluxes.f90_  is already multiplied by negative as prescribed in the budget or not? Which would mean F_{lambda} should be computed as F_{lambda} = uabaro-ua2baro+ep1baro just before plotting?

2. I also noticed that F_{phi'} = -<u'v'***A**cos(phi'+phi)> (but should be F_{phi'} = -<u'v'cos(phi'+phi)> ) on https://hn2016falwa.readthedocs.io/en/latest/Object%20Oriented%20Interface.html 

Thank you very much for this wonderful tool, LWA is just amazing!

**Comment by csyhuang** [comment, marked as answer]
Hi @silver-fox-9244 

1. The minus sign is already multiplied into the value. As coded in `oopinterface.py`:
https://github.com/csyhuang/hn2016_falwa/blob/bb95b87aabc082f758a66bd55da5694f6eff8bec/falwa/oopinterface.py#L743-L746

2. Thanks for spotting out a typo. It has just been fixed!

**Reply by silver-fox-9244** [reply]
Thank you very much for the clarity, 'much appreciated!

**Reply by csyhuang** [reply]
Don't forget to leave a star for this repo if you like it! 😉
